### ISSUE TITLE
Support for mobile browsers

### ISSUE THREAD

**Issue by dellagustin** [podStation/podStation#56]
i.e. Kiwi browser

**Comment by dellagustin** [comment]
Hello @ashishpatel1992 @dhara03, if you would like to contribute, this is an issue where I could use your web development experience.

Do you have experience developing reponsive UIs that work on mobile and desktop?

podStation can be run on some mobile browsers (there is a blocker, #202 , but it is solvable). Although it runs, the user experience is terrible, as it was designed for desktop. Any advice here would be welcome on how to make it mobile friendly.

**Comment by dhara03** [comment]
Okay@dellagustin , let me check this and will get back to you.

**Comment by ashishpatel1992** [comment]
@dellagustin Do you think we can use any UI framework like MaterialUI
If yes let me explore the options, if not then we go with traditional css. (But it will be like reinventing the wheel)
What do you say?

**Comment by dellagustin** [comment]
@ashishpatel1992 in the long term I think a full refactor of the UI code will be needed using a framework. At the moment I am looking for some quick wins with CSS trickery that would make it minimally usable on mobile, but really quick wins, as even with good UI on the mobile browsers, there is still some blockers such as #250 , therefore I don't want to dedicate too much effort to it yet.

**Comment by dellagustin** [comment]
Hello @dhara03 , I'm bringing the conversation from https://github.com/kiwibrowser/src/issues/360#issuecomment-846443554 here.

Did you try using the command `chrome.windows.create({url:'/podstation.html'})` on the console, when remote debugging, to open a podStation tab?

**Comment by dhara03** [comment]
Yes @dellagustin , I tried that way. And I also tried the way Ashish Patel has applied in console. But still blank page is coming.
